app/src/ui/lib/filter-list.tsx

```diff
@@ -220,6 +234,20 @@ export class FilterList<T extends IFilterListItem> extends React.Component<
     )
   }
 
+  public selectFirstItem(focus: boolean = false) {
+    if (this.list !== null) {
+      const next = this.list.nextSelectableRow('down', -1)
```

Comment:
i saw that we use `this.list.nextSelectableRow('down', -1)` three times in this file to get the first item. do we want to extract that to something that's easy to recognize? like `firstRow()`